ts/src/management/tunnelManagementHttpClient.ts

```diff
@@ -30,6 +30,32 @@ function comparePorts(a: TunnelPort, b: TunnelPort) {
     return (a.portNumber ?? Number.MAX_SAFE_INTEGER) - (b.portNumber ?? Number.MAX_SAFE_INTEGER);
 }
 
+function parseDate(value?: string | Date) {
+    return typeof value === 'string' ? new Date(Date.parse(value)) : value;
+}
+
+/**
+ * Fixes Tunnel properties of type Date that were deserialized as strings.
+ */
+function parseTunnelDates(tunnel: Tunnel | null) {
```

Comment:
`tunnel?:`

Reply:
No, it should be an object or `null`, never `undefined`.